### DISCUSSION TITLE
Share vpn via hotspot for Non-rooted devices

### DISCUSSION THREAD

**Discussion by loliana232** [Mygod/VPNHotspot#558] [Q&A]
I have a new non-rooted android phone ( oppo reno 10 pro ) for som reasons I just don't wanna root my phone right now. I want to share my vpn connection from my device to an old xbox that doesn't even support proxy. So the inly way for me is to share vpn with it via hotspot. I've tried everything but never found a solution. Can anyone help me ??
( but please don't tell me buy a VPN router )
@Mygod

**Comment by xliszt** [comment]
try this out https://github.com/pyamsoft/tetherfi
